pkg/kubectl/resource_printer.go

```diff
@@ -2527,6 +2531,14 @@ func NewTemplatePrinter(tmpl []byte) (*TemplatePrinter, error) {
 	}, nil
 }
 
+func (p *TemplatePrinter) AllowMissingKeys(allow bool) {
```

Comment:
I can take this bit out if we want to just apply this to the jsonpath printer for starters